src/components/Sidebar/Sidebar.jsx

```diff
@@ -90,36 +93,33 @@ const Sidebar = () => {
                     color: '#32424E',
                   }}
                 >
-                  {index === 1 ? (
+                  {index === 0 ? (
                     <HomeOutlined />
+                  ) : index === 1 ? (
+                    <AccountBalanceWalletOutlined
+                      style={{ color: '#04819E' }}
+                    />
                   ) : index === 2 ? (
-                    <AccountBalanceWalletOutlined />
+                    <AccountCircleOutlined />
                   ) : index === 3 ? (
                     <AccountCircleOutlined />
                   ) : (
-                    <ContactSupportOutlined />
+                    <ContactSupportOutlined  />
                   )}
                 </ListItemIcon>
                 <ListItemText
+                
                   primary={text}
                   disableTypography={true}
                   sx={{ fontWeight: '700', color: '#32424E' }}
+                  style={{
+                    color: text === 'BILLING' ? '#04819E' : 'null',
+                  }}
```

Comment:
```suggestion
                  sx={{ fontWeight: '700', color: text === 'BILLING' ? '#04819E' : '#32424E'  }}         
```
I would not mix these two ways of writing styles. For one, I would avoid settings styles using `style={{...}}` if possible, since this has the highest specificity in the CSS cascade and can only be overwritten by using `!important`. While it may not cause you problems here, you might run into situations where you'll use a third party package that offers you some component with a set of functionality out of the box, and when you try to style it according to your needs, you may run into trouble precisely because some style attributes have higher specificity than you expect (ask me how I know 🥲).

Styles written using the Material UI `sx` prop are applied to Material UI class names, you can see the difference in specificity in your browser developer tools.

![image](https://user-images.githubusercontent.com/38941030/146918987-851a9ea0-2f23-4346-a828-58fcc06f1164.png)